book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
```

Comment:
`many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental` 🔥